samples/net/sockets/echo_client/overlay-ot-rcp-host-uart.conf

```diff
@@ -0,0 +1,49 @@
+CONFIG_REQUIRES_FULL_LIBC=y
```

Comment:
The instructions from commit message 2487f352b9e3ae2a9d546378274986e58581a681 or 5421b134943a19ba97939ff56b05d92561efe74e could be included somewhere in the readme file, I doubt anyone would look for such info in the commit history.